packages/suite/src/components/settings/SettingsSectionItem.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode } from 'react';
+
+import { SectionItem } from 'src/components/suite';
+import { useAnchor } from 'src/hooks/suite/useAnchor';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+
+interface SettingsSectionItemProps {
+    anchorId: SettingsAnchor;
+    children: ReactNode;
+}
+
+export const SettingsSectionItem = ({ anchorId, children }: SettingsSectionItemProps) => {
+    const { anchorRef, shouldHighlight } = useAnchor(SettingsAnchor.PinProtection);
```

Comment:
you probably wanted to use anchorId here

Reply:
239dd1703997060338d34ea7ae49b149e94a033c